ebean-test/src/test/java/org/tests/batchload/TestQueryDisableLazyLoad.java

```diff
@@ -83,7 +82,29 @@ public void onAssocOne_when_partial() {
     Order order = l0.get(0);
 
     // normally invokes lazy loading
-    assertNull(order.getCustomer().getStatus());
+    assertThrows(LazyInitialisationException.class, () -> order.getCustomer().getStatus());
+
+    List<String> loggedSql = LoggedSql.stop();
+    assertThat(loggedSql).hasSize(1);
+  }
+
+  @Test
+  public void onSetter_expect_LazyInitialisationException() {
+    ResetBasicData.reset();
+    LoggedSql.start();
+
+    List<Order> l0 = DB.find(Order.class)
+      .setDisableLazyLoading(true)
+      .select("status, orderDate")
+      .orderBy().asc("id")
+      .findList();
+
+    assertThat(l0).isNotEmpty();
+
+    Order order = l0.get(0);
+
+    // normally invokes lazy loading
+    assertThrows(LazyInitialisationException.class, () -> order.setShipDate(new Date(System.currentTimeMillis())));
```

Comment:
I'm unsure, if this should fail.., but I understand your concern, that this would break changelog etc.
I think I have no use case, so lets's also fail on setter